AssetStudio/AssetsManager.cs

```diff
@@ -231,6 +235,49 @@ private void LoadWebFile(FileReader reader)
             }
         }
 
+        private void LoadZipFile(FileReader reader)
+        {
+            Logger.Info("Loading " + reader.FileName);
+            try
+            {
+                using (ZipArchive archive = new ZipArchive(reader.BaseStream, ZipArchiveMode.Read))
+                {
+                    foreach (ZipArchiveEntry entry in archive.Entries)
+                    {
+                        try
+                        {
+                            string dummyPath = Path.Combine(Path.GetDirectoryName(reader.FullPath), reader.FileName, entry.FullName);
+                            // create a new stream
+                            // - to store the deflated stream in
+                            // - to keep the data for later extraction
+                            Stream streamReader = new MemoryStream();
+                            entry.Open().CopyTo(streamReader);
```

Comment:
The entry stream should be closed after copying.

Reply:
Solved via using.
I simple-mindedly followed some example code for that part - apparently, it was a bad example^^